doc/running.rst

```diff
@@ -155,6 +155,96 @@ Custom templates
+
+.. code-block:: python
+
+   mm = nest.Create('multimeter', params={'record_from': ['I_kernel1__X__spikes1',
+                                                          'I_kernel2__X__spikes2',
+                                                          'I_kernel3__X__spikes3'],
+                                          'interval': .1})
+   nest.Connect(mm, neuron)
+
+The output shows the currents for each synapse (three bottom rows) and the net effect on the membrane potential (top row):
+
+.. figure:: https://raw.githubusercontent.com/nest/nestml/master/doc/fig/nestml-multisynapse-example.png
+   :alt: NESTML multisynapse example waveform traces
+
+For a full example, please see `tests/resources/iaf_psc_exp_multisynapse.nestml <https://github.com/nest/nestml/blob/master/tests/resources/iaf_psc_exp_multisynapse.nestml>`_ for the full model and ``test_multisynapse`` in `tests/nest_tests/nest_multisynapse_test.py <https://github.com/nest/nestml/blob/master/tests/nest_tests/nest_multisynapse_test.py>`_ for the corresponding test harness that produced the figure above.
+
+
+Multiple input ports with vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See :ref:`Multiple input ports with vectors` for an example with input ports defined as vectors.
+
+Each connection in NEST is denoted by a receiver port or ``rport`` number which is an integer that starts with 0. All default connections in NEST have the ``rport`` 0. NESTML routes the spikes with ``excitatory`` and ``inhibitory`` qualifiers into separate input buffers, whereas NEST identifies them with the same ``rport`` number.
+
+During the code generation for NEST, NESTML maintains an internal mapping between NEST ``rports`` and NESTML input ports. A list of port names defined in a model and their corresponding ``rport`` numbers can be queried from the status dictionary using the NEST API. For neurons with multiple input ports, the ``receptor_type`` values in the ``nest.Connect()`` call start from 1 as the default ``receptor_type`` 0 is excluded to avoid any accidental connections.
+
+For the example mentioned :ref:`here <Multiple input ports with vectors>`, the ``receptor_types`` can be queried as shown below:
+
+.. code-block:: python
+
+   neuron = nest.Create("multi_synapse_vectors")
+   receptor_types = nest.GetStatus(neuron, "receptor_types")
+
+The name of the receptors of the input ports are denoted by suffixing the ``vector index + 1`` to the port name. For instance, the receptor name for ``foo[0]`` would be ``FOO_1``.
+
+The above code querying for ``receptor_types`` gives a list of port names and NEST ``rport`` numbers as shown below:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Input port name
+     - NEST ``rport``
+   * - AMPA_spikes
+     - 0
```

Comment:
I'm still confused; I thought NEST started counting at 1 for "multisynapse" neurons. That's also what is described here:

https://github.com/nest/nestml/pull/849/files#diff-cca67b57fc552badda2b746e8ee77ea760772e475caa48ccc820d40d15f2ee1bR205